libpod/runtime_img.go

```diff
@@ -9,6 +9,7 @@ import (
 	"net/url"
 	"os"
 
+	buildah "github.com/containers/buildah/define"
```

Comment:
Is `github.com/containers/podman/v3/libpod/define` going away eventually?  If not, rather than buildah here and just define elsewhere, I think this should be `bdefine` or something that's easily identified as being different than Podman's define.

Reply:
I concur with Tom.  I would prefer to alias _all_ imports of this package as `buildahDefine`.  My eyes are used to read `define` as the Podman package.

Reply:
I am waiting for my other pull-never  gets merged, before fixing this, in case of conflicts.

Reply:
Done.